docs/src/main/asciidoc/security-overview-concept.adoc

```diff
@@ -1,68 +1,67 @@
 [id="security-overview-concept"]
 = Quarkus Security overview
 include::_attributes.adoc[]
-:categories: security
+:categories: security,getting-started
```

Comment:
This would be a good guide for someone new to Quarkus also as it introduces and connects all things  Security.

I added this category but I'm having de ja vous. Is there a reason we didn't add this before?

Reply:
@michelle-purcell I did not think about it, the main reason behind it, good idea indeed to let users find a security overview doc on their getting started path, thanks